doc/whats_new/v0.22.rst

```diff
@@ -60,6 +60,13 @@ Changelog
 - |Fix| :func:`utils.check_array` now correctly converts pandas DataFrame with
   boolean columns to floats. :pr:`15797` by `Thomas Fan`_.
 
+- |Fix| :func:`utils.check_is_fifted` accepts back an explicit ``attributes``
+  argument to check for specific attributes as explicit markers of a fitted
```

Comment:
``attributes`` and ``all_or_any``

Reply:
> The `attributes` argument to check for specific attributes. 

I think that's correct syntax highlighting, there should be no need to write each occurrence of attributes as `attribute`.

Reply:
OK, I see your point, added a mention about ``all_or_any`` to what's new.